docs/release_data.md

```diff
@@ -0,0 +1,361 @@
+| pytorch   | opt_1.3b      | NeelNanda/pile-10k | 34.07                    | 57.05%            | 19.92                    | 57.89% | 1.71 | \-1.44% |
+| pytorch   | bloom_1b7     | NeelNanda/pile-10k | 29.74                    | 49.95%            | 13.3                     | 46.34% | 2.24 | 7.79% |
+| pytorch   | bloom_7b1     | NeelNanda/pile-10k | 12.36                    | 60.14%            | 3.22                     | 57.64% | 3.83 | 4.34% |
+| pytorch   | opt_2.7b      | NeelNanda/pile-10k | 23.19                    | 63.67%            | 12.24                    | 63.65% | 1.89 | 0.03% |
+| pytorch   | opt_6.7b      | NeelNanda/pile-10k | 13.5                     | 67.01%            | 4.1                      | 67.69% | 3.29 | \-1.00% |
+| pytorch   | llama_13b     | NeelNanda/pile-10k | 8.9                      | 56.88%            | 2.24                     | 76.27% | 3.97 | \-25.42% |
+| pytorch   | llama_7b      | NeelNanda/pile-10k | 13.56                    | 58.55%            | 4.4                      | 73.61% | 3.09 | \-20.46% |
+| pytorch   | gpt_j_6b      | NeelNanda/pile-10k | 10.76                    | 67.59%            | 4.38                     | 68.31% | 2.46 | \-1.05% |
+| pytorch   | flan_t5_large | samsum             | 69.75                    | 46.25 (rougeLsum) | 33.16                    | 47.67 (rougeLsum) | 2.1 | \-2.99% |
+| pytorch   | gpt_neox_clm  | wikitext           | 1.47                     | 4.04 (eval_loss)  | 0.65                     | 3.52 (eval_loss) | 2.27 | \-14.78% |
+| pytorch   | gpt_j_6b_clm  | wikitext           | 0.86                     | 3 (eval_loss)     | 0.28                     | 2.34 (eval_loss) | 3.1 | \-28.67% |
+| onnx      | whisper_large | lambda-openai      | 2.11                     | 97.07%            | 1.13                     | 96.96% | 1.87 | 0.12% |
+
+
+
+## LLM Runtime Inference based on Pytorch
+Environment:
+
+Pytorch: 2.0.1+cpu
+
+### LLMs
+
+
+| Framework | Model                 | Input    | Output | Steps    | INT8      | FP32      | BF16      | FP8       | INT8/FP32 | BF16/FP32 | FP8/FP32 |
+| --------- | --------------------- | -------- | ------ | -------- | --------- | --------- | --------- | --------- | --------- | --------- | -------- |
+| pytorch   | gpt-neox-20b          | 32       | 32     |          | 9283 (ms) |           |           |           |           |           |          |
+| pytorch   | dolly-v2-3b           | 32       | 32     |          | 3191 (ms) | 3798 (ms) | 2689 (ms) | 1.19x     | 1.41x     |           |
+| pytorch   | stable_diffusion_v2_1 | 512\*512 | 20     |          | 16.98 (s) | 2.83 (s)  |           | 6.00x     |           |
+| pytorch   | Steps = 20            |          |        |          |           |           |           |           |           |           |
+| pytorch   | stable_diffusion_v1_5 | 512\*512 | 20     | 2.18 (s) | 10.94 (s) | 2.74 (s)  | 5.01x     | 3.99x     |           |
```

Comment:
stable diffusion is not llm strictly speaking, making the table not unified. 

move to Stable Diffusion section with accuracy table and performance table